l10n.html

```diff
@@ -0,0 +1,661 @@
+            <form name="option">
+                <div id="option" class="option">
+                    <!-- add more options here -->
+                    <div id="diff-option" class="column">
+                        <strong>Compare</strong>
+                        <hr noshade="">
+                        <label for="diff-notodo"
+                            title="Ignore TODO marks, only reports addition/deletion.">
+                            <input type="checkbox" id="diff-notodo" name="diff-notodo"> Igore TODO mark
+                        </label>
+                    </div>
+                    <div id="gen-option" class="column">
+                        <strong>New</strong>
+                        <hr noshade="">
+                        <label for="gen-keepnew" 
+                            title="If target is newer than base, don't remove entries only exist in target locale.">
+                            <input type="checkbox" id="gen-keepnew" name="gen-keepnew"> Keep newer entries
+                        </label>
+                        <label for="gen-keepall" 
+                            title="Don't remove any entries in target locale.">
+                            <input type="checkbox" id="gen-keepall" name="gen-keepall"> Keep all entries
+                        </label>
+                        <label for="gen-nocomment"
+                            title="Don't include comment in generated lang-xx.js.">
+                            <input type="checkbox" id="gen-nocomment" name="gen-nocomment"> No comments
+                        </label>
+                    </div>
+                    <div id="lint-option" class="column">
+                        <strong>Check</strong>
+                        <hr noshade="">
+                        <label for="lint-enpunct"
+                            title="When checking translation, don't warn for untranslated punctuations.">
+                            <input type="checkbox" id="lint-enpunct" name="lint-enpunct"> Allow English punctuation
+                        </label>
+                    </div>
+                </div>
+            </form>
+            <div id="log" class="panel log"></div>
+            <textarea id="result" cols="100" rows="40" readonly=""></textarea>
+        </div>
+        <!-- // script tags holds lang-xx.js will be dynamically created //
+            <script id="baseDict" type="text/javascript"></script>
+            <script id="targetDict" type="text/javascript"></script>
+        -->
+        <script type="text/javascript">
+            var modules = {}; // for locale.js
+        </script>
+        <script type="text/javascript" src="locale.js"></script>
+        <script type="text/javascript">
+        (function () {
+            // Space separated puncutation list (same order as 'en'), used by lint().
+            // If one puncuation can have multiple translations, just group them together.
+            SnapTranslator.punctuations = {
```

Comment:
I didn't put them into translation file mainly because it's not part of translation and completely useless at runtime.

There's a chance the linting rules grow to be more complex in future, you might want to keep it separately so it doesn't pollute the translation file a lot.